Added ISIC Archive, CaPTk

awanafiaz opened the pull request:
Added new entries in resource (ISIC Archive) and Imaging (CaPTk)

awanafiaz committed:
Added ISIC Archive, CaPTk

Added new entries in resource (ISIC Archive) and Imaging (CaPTk)

carriewright11 commented on alldataFilesEditor.Rmd:
```diff
@@ -30,21 +30,58 @@ resource %<>%
+  Name = "The Cancer Imaging Phenomics Toolkit (CaPTk)",
+  `Link (Hyperlinked Over Name)` = "https://www.med.upenn.edu/cbica/captk/",
+  Subcategories = "Radiomics", 
+  `Unique Factor` = "CaPTk beyond being a stand-alone software client it also makes its computational functionality on image analytics available for free, through the Image Processing Portal (IPP) that offers compute power through the CaPTk's supporting high performance computing cluster.",
```
```suggestion
  `Unique Factor` = "CaPTk is a stand-alone software client that allows for free image analysis through the Image Processing Portal (IPP) that offers compute power through CaPTk's supporting high performance computing cluster.",
```

carriewright11 replied:
just simplifying a bit

carriewright11 commented on imagingTable.Rmd:
```diff
@@ -29,8 +29,8 @@ library(magrittr)
 # Load file for Imaging Table
-generalData <- read_csv(file = here::here("data/ictr_table_imaging.csv"))
-generalData
+generalData <- read_csv(file = here::here("data/ictr_table_imaging_new.csv")) 
```
what is the difference between ictr_table_imaging.csv and ictr_table_imaginag_new.csv?

Also the letters got rearranged here - we could rename it using git mv oldname newname

awanafiaz replied:
The _ictr_table_imaging.csv_ is the original file. The _new is the csv file with the new entries. Overwriting the original file did not work since it kept adding the same row (new tool information) every time it compiled the Rmd file.

carriewright11 commented:
Looks good! I just had a question about the file names - want to be clear what is what - we could add this to a contributing.md file to help us in the future.  We can also name files in a way that might make it more clear what a file contains. I had one suggestion that you can commit with the commit suggestion button if you like it - just simplifying some of the language - I think it is from the tool creator, I just want the unique factors to be concise.

awanafiaz committed:
Update alldataFilesEditor.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz commented:
> Looks good! I just had a question about the file names - want to be clear what is what - we could add this to a contributing.md file to help us in the future. We can also name files in a way that might make it more clear what a file contains. I had one suggestion that you can commit with the commit suggestion button if you like it - just simplifying some of the language - I think it is from the tool creator, I just want the unique factors to be concise.

Response: There are basically 3 version of each csv file. 
1. filename.csv = The original files that we had.
2. filename_new.csv = the new entries/modifications added to the original files. Such changes were performed in the `alldataFilesEditor.Rmd`.  (Overwriting the original file did not work since it kept adding the same row with new tool information every time it compiled the Rmd file.) 
3. modifiled_filename.csv = the data files after preparing them for website view (keeping certain columns, removing others, hyperlinked, etc.). These were compiled in each of the different table and then we used the modified files (with all the editing) in the "All platforms" page. This allowed us to avoid copy pasting the same wrangling codes again in the "allTables.Rmd". 

The workflow is basically this: 
1. We add new/modify new/old entries in each datasets in the `alldataFilesEditor.Rmd` (filename.csv > filename_new.csv)
2. We wrangle each dataset and generate the modified datasets suitable for using with `DT::dataTable()` (filename_new.csv > modified_filename.csv).

awanafiaz commented:
> I had one suggestion that you can commit with the commit suggestion button if you like it - just simplifying some of the language - I think it is from the tool creator, I just want the unique factors to be concise.

Gotcha. I accepted your commit. I also copied the owner's response from the Google form response. But I see that you opened an issue for that tool which had your version of the Unique Factor. It is fine for me as long as you and the tool creator/developer is okay with the description.

awanafiaz committed:
added platforms definiton, + contributing.md

1. Added the platform and ITCR platform definitions in the platformsOverview.Rmd 

2. added contributing.md file explaining the file name structure and how each file is connected with the other.
